lib/package_utils.rb

```diff
@@ -23,7 +24,7 @@ def self.incompatible_reason(pkg)
 
   def self.get_url(pkg, build_from_source: false)
     if !build_from_source && pkg.binary_sha256&.key?(ARCH.to_sym)
-      return "https://gitlab.com/api/v4/projects/26210301/packages/generic/#{pkg.name}/#{pkg.version}_#{ARCH}/#{pkg.name}-#{pkg.version}-chromeos-#{ARCH}.#{pkg.binary_compression}"
+      get_binary_url(pkg)
```

Comment:
`return get_binary_url(pkg)`?

Reply:
There's no need for a return there, as it will return that value anyways, but I can add one if you'd prefer that.

Reply:
Probably should for clarity and consistency.  The elsif line below uses a return.